src/ng/directive/input.js

```diff
@@ -1100,7 +1100,7 @@ var inputType = {
    *                  Can be interpolated.
    * @param {string=} step Sets the `step` validation to ensure that the value entered matches the `step`
    *                  Can be interpolated.
-   * @param {string=} ngChange AngularJS expression to be executed when the ngModel value changes due
+   * @param {expression=} ngChange AngularJS expression to be executed when the ngModel value changes due
    *                  to user interaction with the input element.
```

Comment:
Nit: Can you align this line right under `ngChange` (to be consistent with the rest)?

Reply:
done